godot-core/src/docs.rs

```diff
@@ -7,6 +7,7 @@
 use crate::registry::plugin::PluginItem;
 use std::collections::HashMap;
 
+#[doc(hidden)]
```

Comment:
Is this necessary? If I check [latest API docs](https://godot-rust.github.io/docs/gdext/master/godot/?search=StructDocs), I don't find the symbol.

If it's indeed necessary, attributes should come after the `///` comment block.